README.md

```diff
@@ -12,6 +12,9 @@ The integration is usable with Kentico Xperience 13 **hotfix 13.0.142** or newer
 ## Installation
 
 1. Install the [Kentico.Xperience.OpenAI.Azure.KX13](https://www.nuget.org/packages/Kentico.Xperience.OpenAI.Azure.KX13) NuGet package in the administration project.
+    - If you encounter an *Unable to resolve dependency 'Azure.AI.OpenAI'...* error during installation, consider a console installation through **Package Manager Console** (*Tools* → *NuGet Package Manager* → *Package Manager Console* in Visual Studio) and install the package with the **-IncludePrerelease** flag:
```

Comment:
- "consider a console installation" sounds a bit as though I should consider the installation of some console, which is not what I want to do -> an idea how to improve: "... try installing the package via the **Package Manager Console**"
- "(*Tools* → *NuGet Package Manager* → *Package Manager Console* in Visual Studio)" -> the 'Package Manager Console' is redundant here, as this is what I am told to look for right before the parentheses start. Also, use bold instead of italics as these are UI elements (check https://kentico.atlassian.net/wiki/spaces/XCE/pages/3160476114/Correct+typography). Consider rewriting to "(under **Tools -> NuGet Package Manager** in Visual Studio)